library/Zend/Form/Annotation/AnnotationBuilder.php

```diff
@@ -383,6 +389,18 @@ protected function checkForExclude($annotations)
     }
 
     /**
+     * @return \Zend\Code\Annotation\Parser\DoctrineAnnotationParser
+     */
+    public function getAnnotationParser()
+    {
+        if (null === $this->annotationParser) {
+            $this->annotationParser = new Parser\DoctrineAnnotationParser();
+        }
+
+        return $this->annotationParser;
+    }
```

Comment:
@carnage I missed that it was being used in the factory as well, and, as such, is part of the public API you need to expose. We can keep it as-is.